source/shared/_cookie_policy_script.html.erb

```diff
@@ -0,0 +1,31 @@
+<body onload="checkCookie()"></body>
```

Comment:
This would mean we end up with 2x `<body>` tags on the page. My understanding was that there should only ever be one.

Reply:
Could you save it as a `.js` file and use `javascript_include_tag` rather than an `.html.erb` partial? See https://middlemanapp.com/basics/helper-methods/#asset-helpers